code/ai/aibig.cpp

```diff
@@ -457,13 +456,46 @@ int ai_big_maybe_follow_subsys_path(int do_dot_check)
 		if ( aip->path_length > 0 ) {
 			int path_done=0;
 			int in_view=0;
+			int default_dist = 5;
```

Comment:
Since `default_dist` never actually changes, put it in the header file and make it a `#define` or a `const`.

Reply:
If it's a const int I thought that this line would then not work?
`aip->path_goal_dist = Default_subsystem_path_pt_dist;`

Reply:
I see nothing wrong with that.  Why do you think it wouldn't work?

If you mean `const` vs. `non-const`, you're working with values here, not pointers.  Since the value is copied, the original variable won't be affected if `aip->path_goal_dist` subsequently changes.